settings.py

```diff
@@ -136,6 +136,10 @@
 # Validation key we need to send in POST response.
 INBOUND_EMAIL_VALIDATION_KEY = 'totally-unsecure-validation-string'
 
+# Sectools
+CUSTOMS_API_URL = 'http://host.docker.internal:10101/'
```

Comment:
I assume this would be https for dev/stage/prod, right ?

Reply:
As mentioned on IRC this doesn't work for me with Linux as the host. We need a more robust solution.

Reply:
> I assume this would be https for dev/stage/prod, right ?

Yes

> As mentioned on IRC this doesn't work for me with Linux as the host. We need a more robust solution.

😢 working on it